src/sass/_classes.scss

```diff
@@ -0,0 +1,27 @@
+$classes: (
```

Comment:
File is now named as _classes.scss and variable also named as $classes, since it holds only layout things: how about moving this into _layout.scss and naming just $layoutClasses or something?